src/test/java/org/opensearch/security/system_indices/SystemIndicesTests.java

```diff
@@ -269,29 +311,45 @@ public void testDelete() throws Exception {
     }
 
     @Test
-    public void testDeleteWithSystemIndices() throws Exception {
+    public void testDeleteDocWithSystemIndicesShouldSucceedAsSuperAdmin() throws Exception {
```

Comment:
NIT: I like the informative name but I think this will be a bit long for consoling logging. You will get the full path `src/test/java/org/opensearch/security/system_indices/SystemIndicesTests.testDeleteDocWithSystemIndicesShouldSucceedAsSuperAdmin` which is pretty long...